cmd/user.go

```diff
@@ -0,0 +1,125 @@
+package cmd
+
+import (
+	"bytes"
+	"encoding/base64"
+	"github.com/cloudflare/cfssl/log"
+	"github.com/k0sproject/k0s/pkg/certificate"
+	"github.com/k0sproject/k0s/pkg/constant"
+	"github.com/pkg/errors"
+	"github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+
+	"html/template"
+	"io/ioutil"
+	"os"
+	"path"
+)
+
+func init() {
+	userCreateCmd.Flags().StringVar(&groups, "groups", "", "Specify groups")
+
+	userCmd.AddCommand(userCreateCmd)
+}
+
+var (
+	groups string
+
+	userKubeconfigTemplate = template.Must(template.New("kubeconfig").Parse(`
+apiVersion: v1
+clusters:
+- cluster:
+    server: {{.JoinURL}}
+    certificate-authority-data: {{.CACert}}
+  name: k0s
+contexts:
+- context:
+    cluster: k0s
+    user: {{.User}}
+  name: k0s
+current-context: k0s
+kind: Config
+preferences: {}
+users:
+- name: {{.User}}
+  user:
+	client-certificate-data: {{.ClientCert}}
+	client-key-data: {{.ClientKey}}
+`))
+
+	// userCmd creates new certs and kubeConfig for a user
+	userCmd = &cobra.Command{
+		Use:   "user",
+		Short: "Manage user access",
+		RunE: func(cmd *cobra.Command, args []string) error {
+			return nil
+		},
+	}
+
+	userCreateCmd = &cobra.Command{
+		Use:   "create [username]",
+		Short: "Create a kubeconfig for a user.",
+		Long:  "Create a kubeconfig with a signed certificate and public key for a given user (and optionally user groups).\nNOTE: A certificate once signed cannot be revoked for a particular user.",
```

Comment:
command line description should not end with a `.`

Reply:
Also I would add an example here too